packages/modal-ui-js/src/lib/components/WalletConnectionFailed.ts

```diff
@@ -30,9 +30,9 @@ export async function renderWalletConnectionFailed(
               <div class="error-wrapper">
                 <div class="error"><svg width="24" height="24" viewBox="0 0 24 24" fill="none"
                     xmlns="http:/*www.w3.org/2000/svg">*/ <rect width="24" height="24" rx="12" fill="#CE5A6F"></rect>
-                    <path d="M17.25 6.75L6.75 17.25" stroke="#141414" stroke-width="1.5" stroke-linecap="round"
+                    <path d="M17.25 6.75L6.75 17.25" stroke="currentColor" stroke-width="1.5" stroke-linecap="round"
                       stroke-linejoin="round"></path>
-                    <path d="M6.75 6.75L17.25 17.25" stroke="#141414" stroke-width="1.5" stroke-linecap="round"
+                    <path d="M6.75 6.75L17.25 17.25" stroke="currentColor" stroke-width="1.5" stroke-linecap="round"
                       stroke-linejoin="round"></path>
                   </svg>Connection Failed</div>
```

Comment:
The "Connection failed" text in the `modal-ui` react seems like the text is blue to fix it there maybe just remove the [wrapping span](https://github.com/near/wallet-selector/blob/modal-responsive-fixes/packages/modal-ui/src/lib/components/ConnectionResult.tsx#L45) element of this text and it will align with the `modal-ui-js` too.

Reply:
fixed